src/routes/meta-info.ts

```diff
@@ -1,8 +1,7 @@
 import { RouteInitEnvironment, RouteRequest } from '../shared/env';
-import { readFileSync } from 'fs';
-import { packageRootPath } from '../shared/rootpath';
-import { join as joinPaths } from 'path';
 import * as os from 'os';
+//@ts-ignore
```

Comment:
please add a comment about why the ts-ignore is necessary here!

Reply:
Done! And also fixed ESM builds requiring import assertion.

Reply:
https://github.com/appy-one/acebase-server/pull/56/commits/e7db94412c9794d93085c302f801bfe687e9a009